bin/yk-config

```diff
@@ -90,7 +90,7 @@ handle_arg() {
             # Use lld.
             OUTPUT="${OUTPUT} -fuse-ld=lld"
             # Embed LLVM bitcode as late as possible.
-            OUTPUT="${OUTPUT} -Wl,--mllvm=--embed-bitcode-final"
+            OUTPUT="${OUTPUT} -Wl,--mllvm=--embed-bitcode-final -Wl,--mllvm=--emulated-tls"
```

Comment:
We should put this flag on its own line and add a comment what it does: forces use of emulated TLS since our JIT doesn't support native TLS.

Reply:
done 👉 e55dc233f3affad262494786a9e04cdc4bcecb38